.github/ISSUE_TEMPLATE/service-proposal.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Service proposal
+about: Template to use to submit a service proposal
+title: "[SERVICE]"
+labels: service
+assignees: ''
+
+---
+
+<!-- 
+
+Use the following template when submitting proposals for new CUSTOM services to this repository.
+
+ Linking to a standard characteristic/service specification document on the Bluetooth SIG's website is sufficient for standard characteristic/service proposals. When proposing standard services, also include specific requirement information for the Mbed BLE implementation of this service (eg: memory constraints, baremetal support, etc) 
+
+The following site can be used to generate a long-format UUID for custom services/characteristics: https://www.uuidgenerator.net/
+This site is useful for generating markdown-formatted tables: https://www.tablesgenerator.com/markdown_tables
+-->
+
+### BLE Standard Service: Yes/No
+### Proposed UUID(s) or link to specification document:
+
+| Svc Name                 | Char Name         | UUID   | Qualifier | Mandatory Props | Optional Props | Security |
```

Comment:
I wouldn't know how to fill out this table. Maybe all this needs is a simple list:
service name - uuid
characteristic name - uuid - mandatory/optional

Reply:
I'm on the fence about whether the table needs all these fields. I copied the same format that the Bluetooth SIG uses in their service/characteristic specification documents. 

It may be overkill but I can see where each field would be useful. It also helps users think in more detail about how their service operates -- eg: _"what kind of security requirements should this service have?"_, and so on.

A list would probably be easier for users to format as well...